### PR TITLE
Unstable test upgrades

### PR DESCRIPTION
**Pull request by MaciejTe**
test(unstable): Mark test_upgrades as xfail

Changelog: None
Ticket: QA-451
Signed-off-by: Maciej Tomczuk <maciej.tomczuk@northern.tech>

### CODE REVIEW

File: tests/tests/test_access.py [hunk 1]
```diff
@@ -239,6 +239,7 @@ def test_initial_restrictions(self, docker_env):
         not bool(os.environ.get("STRIPE_API_KEY")),
         reason="STRIPE_API_KEY not provided",
     )
+    @pytest.mark.xfail(reason="QA-451")
```
[the remark this answers is not in the document]
Comment (MaciejTe) [reply tests/tests/test_access.py:242]: No, if test case passes, pipeline won't fail. Basically, `xfail` marks test case as unstable. The pipeline won't fail if this TC fails

[the remark this answers is not in the document]
Comment (oleorhagen) [reply tests/tests/test_access.py:242]: Oh, I misread it. It is marked as `xpass`

**Review by oleorhagen — CHANGES_REQUESTED** [review]
I don't think this is the way we should do it tbh.

1. I don't think we can simply mark this and close the ticket. The problem is still there.

**Comment by MaciejTe** [comment]
As agreed during standup, I'm closing this task since it will be moved to backend integration tests.

**Pull request closed without merging**